src/ray/gcs/gcs_worker_manager.h

```diff
@@ -61,6 +64,12 @@ class GcsWorkerManager : public rpc::WorkerInfoGcsServiceHandler {
       rpc::UpdateWorkerNumPausedThreadsReply *reply,
       rpc::SendReplyCallback send_reply_callback) override;
 
+  /// Initialize with the gcs tables data synchronously.
```

Comment:
A somewhat related comment: I think we shouldn't have gcs_worker_manager in the first place. GCS manages nodes (raylets) and nodes manage worker processes so GCS is never a source of truth for worker processes. If someone wants to know about a worker, they should ask the corresponding raylet directly.